WalletWasabi.Gui/Controls/WalletExplorer/TransactionViewerViewModel.cs

```diff
@@ -73,7 +73,16 @@ public TransactionViewerViewModel(WalletViewModel walletViewModel) : base("Trans
 						Title = "Export Binary PSBT"
 					};
 
-					if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+					if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+					{
+						var initailDirectory = Path.Combine("/media", Environment.UserName);;
```

Comment:
nit: 2 semicolons

Reply:
Also rename `initailDirectory` to `initialDirectory`